switchmap/server/db/attributes.py

```
@@ -403,6 +403,21 @@ class DeviceAttribute:
     ts_created = graphene.DateTime(description="Row Creation Timestamp")
 
 
+class DeviceMetricsAttribute:
+    """Descriptive attributes for DeviceMetricsHistory table."""
+
+    hostname = graphene.String(
+        resolver=lambda obj, info: (
```

Ruff (0.12.2)

Unused lambda argument: `info`

(ARG005)